qcodes_contrib_drivers/drivers/BlueFors/BlueFors.py

```diff
@@ -0,0 +1,200 @@
+# This Python file uses the following encoding: utf-8
+# Etienne Dumur <etienne.dumur@gmail.com>, september 2020
+
+import os
+import pandas as pd
+from datetime import date
+from qcodes.instrument.base import Instrument
+
+class BlueFors(Instrument):
+    """
+    This is the QCoDeS python driver to extract the temperature and pressure
+    from a BlueFors fridge
+    """
+
+    def __init__(self, name                      : str,
+                       folder_path               : str,
+                       channel_vacuum_can        : int=None,
+                       channel_pumping_line      : int=None,
+                       channel_compressor_outlet : int=None,
+                       channel_compressor_inlet  : int=None,
+                       channel_mixture_tank      : int=None,
+                       channel_venting_line      : int=None,
+                       channel_50k_plate         : int=None,
+                       channel_4k_plate          : int=None,
+                       channel_magnet            : int=None,
+                       channel_still             : int=None,
+                       channel_mixing_chamber    : int=None,
```

Comment:
these can't be None by default because if they are None then the get_pressure method will not work, right? so either the driver should to NOT create (or expose) the pressure parameters for those channels or get_pressure method should be made smarter to raise a meaninful exception if None is being passed to it.

Reply:
Hum I am not sure I am following your comment.
However seeing the driver now after some time I recognize that this is weird.
The main purpose was to allow a driver to being instanced without magnet, which is an option.
I could put all channel mandatory and only the magnet as optional, what do you think?

Reply:
sorry, this is my bad, i misread the `if channel_magnet is not None` before every add_parameter call, all is good.

> The main purpose was to allow a driver to being instanced without magnet, which is an option.
I could put all channel mandatory and only the magnet as optional, what do you think?

if it makes sense to also use the driver without mixing_chamber and 4k_plate, etc etc - then yes. if it only makes sense to use the driver with/without magnet ONLY, then only the magnet channel should be optional.